compact/LUXETracker.xml

```diff
@@ -45,13 +45,23 @@
 
   <readouts>
     <readout name="SiHits">
-      <segmentation type="CartesianGridXY" grid_size_x="0.025*mm" grid_size_y="0.025*mm"/>
-      <id>system:1,layer:2,stave:3,sensor:7,x:32:-16,y:-16</id>
+      <id>${GlobalTrackerReadoutID}</id>
```

Comment:
Do we not need the `segmentation` here? Not sure because at least simulation still seems to be working.

Reply:
I think we'd probably need it when we have proper digitisation and pixel clustering. But for now, the digitisation processor doesn't find the surface if the cellID includes x and y (no surface found for cellID : system:1,side:0,layer:0,module:0,sensor:5,x:41,y:-30)

Reply:
I just tried this locally with the `segmentation` in and the current definition of the `GlobalTrackerReadoutID`, and I could run this in digitization. I did however, run this only with the particle gun. Maybe I need to also test with ptarmigan inputs to trigger the error?

Reply:
Thanks for checking, I also can't reproduce the error now, so I'm adding back the segmentation.